test/Microsoft.AspNet.Mvc.Core.Test/ActionResults/CreatedAtActionResultTests.cs

```diff
@@ -85,16 +84,21 @@ private static ActionContext GetActionContext(HttpContext httpContext)
                                     new ActionDescriptor());
         }
 
-        private static HttpContext GetHttpContext(HttpResponse response)
+        private static HttpContext GetHttpContext()
         {
             var httpContext = new Mock<HttpContext>();
-
+            var realContext = new DefaultHttpContext();
```

Comment:
Might be cleaner to simply return the `realContext` instance. All it would require is mocking the `Request.RequestServices` service provider.